spmd/tensor/parallel/api.py

```diff
@@ -74,6 +74,10 @@ def parallelize_module(  # type: ignore[return]
         device_mesh = _create_1d_device_mesh(device_mesh, tp_mesh_dim)
 
     if isinstance(parallelize_plan, ParallelStyle):
+        # RowwiseParallel or ColwiseParallel
+        if not isinstance(parallelize_plan, PairwiseParallel):
```

Comment:
We should do explicitly check IMO for Rowwise and Colwise

Reply:
OK